rskj-core/src/test/java/co/rsk/peg/federation/FederationTestUtils.java

```diff
@@ -31,14 +31,22 @@
 import co.rsk.bitcoinj.script.Script;
 import co.rsk.bitcoinj.script.ScriptBuilder;
 import java.math.BigInteger;
+import java.time.Instant;
 import java.time.ZonedDateTime;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 import java.util.stream.Collectors;
+
+import co.rsk.peg.constants.BridgeConstants;
 import org.ethereum.crypto.ECKey;
 
-public class FederationTestUtils {
+public final class FederationTestUtils {
+
+    public static final long GENESIS_FEDERATION_CREATION_BLOCK_NUMBER = 1L;
```

Comment:
It could be used everywhere `FederationArgs` is called since when you use its constructor you need to put the `creationBlockNumber`. `FederationArgs` is used too much. Let me know if that makes sense or if we should change the constant name.

Reply:
But it's only used to build the genesis federation, which is done inside this class. Are you building the genesis federation anywhere else using this constant?

Reply:
Should I remove the prefix GENESIS_FEDERATION? This helps to reuse the value. or do you think is better to put it as a `private` constant in the class/method?

Reply:
I think it should simply go inside getGenesisFederation method. Where else is it being used? This is the only place where the genesis federation is built, isn't it?

Reply:
Done! GENESIS_FEDERATION_CREATION_BLOCK_NUMBER was removed as class level constant and was included in the only method it is being used.

Reply:
Thanks!